client/src/Components/WorkCard/WorkCard.tsx

```diff
@@ -1,44 +1,64 @@
-import React from 'react';
-import { Card } from 'antd';
+import React, { useState } from 'react';
+import {
+  Card, message, Popconfirm, Modal,
+} from 'antd';
+import { EditOutlined, DeleteOutlined } from '@ant-design/icons';
+import { useTranslation } from 'react-i18next';
+import { WorkCardProps } from '../../utils';
 import './style.css';
-import { EditOutlined, SettingOutlined } from '@ant-design/icons';
 
 const { Meta } = Card;
-interface WorkCardProps {
-  work: {
-    title: string;
-    content: string;
-    image: string;
-    userId: number;
-  };
-  actions: {
-    edit: () => void;
-    setting: () => void;
-  };
-  isAuth: {
-    isAuth: boolean;
-  }
-}
 
 function WorkCard({ work, actions, isAuth }: WorkCardProps) {
-  return (
+  const { t } = useTranslation();
+  const [isClickEdit, setIsClickEdit] = useState(false);
 
-    <Card
-      className="work-card"
-      hoverable
-      actions={isAuth.isAuth ? [
-        <SettingOutlined onClick={actions?.edit} key="setting" />,
-        <EditOutlined onClick={actions?.setting} key="edit" />,
-      ] : []}
-      cover={<img alt="example" src={work.image} />}
-    >
-      <Meta
-        className="work-card-body"
-        title={work.title}
-        description={work.content}
-      />
-    </Card>
+  const handelDelete = () => {
+    actions.delete(work.id);
+    const key = 'deleted';
+    message.loading({ content: t('successfully-delete-load'), key });
+    setTimeout(() => {
+      message.success({ content: t('successfully-delete'), key, duration: 2 });
+    }, 1000);
+  };
```

Comment:
```suggestion

  const handelDelete = async () => {
    try {
      // message.loading({ content: t('successfully-delete-load') });
      const data = await actions.delete(work.id);
      message.success({ content: t('successfully-delete'), key, duration: 2 });
    } catch (err) {
      console.log(err.data.msg, 'err');
      message.error({ content: t('successfully-delete') });
    }
  };
```
try to handle both success and faliure cases